GX-TimeFrequency/src/minimax_grids.F90

```diff
@@ -27,9 +29,35 @@ module minimax_grids
   !> Main entry point for client code.
   public :: gx_minimax_grid, gx_minimax_grid_frequency
 
+  !> Declare interfaces for BLAS/LAPACK subroutines
```

Comment:
blas and lapack interfaces have nothing to do with minimax, and should be in their own module. I'd be happy enough for them to be placed in a module, in gx-common lib.

I'd also recommend using preexisting interfaces. See this discussion on[ fortran lang](https://fortran-lang.discourse.group/t/lapack-interfaces/2462/14) for some references. Or take exciting's: Benedikt wrote several. W.r.t. licensing I think they're fine to take (of course, would be good to reference him in the code). See
`exciting/src/lapack_wrappers/f95_interfaces`

Reply:
Yes, they should go in gx-common. I added them because it had to be done, there were a lot of lingering warnings because of that. I will take care.

Reply:
Sure, these interfaces can improve memory performance. My points were:

- It's in the wrong place
- Other people have written them many times over - option copy and paste

Reply:
I got them from one of the links in the Fortran discourse, the one from Ondřej